src/Illuminate/Database/Schema/SqliteSchemaState.php

```diff
@@ -16,16 +16,14 @@ class SqliteSchemaState extends SchemaState
     public function dump(Connection $connection, $path)
     {
         with($process = $this->makeProcess(
-            $this->baseCommand().' .schema'
+            $this->baseCommand().' ".schema --indent"'
         ))->setTimeout(null)->mustRun(null, array_merge($this->baseVariables($this->connection->getConfig()), [
             //
         ]));
 
-        $migrations = collect(preg_split("/\r\n|\n|\r/", $process->getOutput()))->reject(function ($line) {
-            return str_starts_with($line, 'CREATE TABLE sqlite_');
-        })->all();
+        $migrations = preg_replace('/CREATE TABLE sqlite_.+\);[\r\n]+/is', '', $process->getOutput());
```

Comment:
The change here seems to have affected an application I have in which it used to return all the tables I had in my sqlite database file, but now only returns the `migrations` one. I'm not sure how to fix it to make it work with all of them though. I think this test is only testing the sqlite in memory and not the sqlite database file and so it got missed.